genesis/genesis_fuji.go

```diff
@@ -46,6 +48,12 @@ var (
 				MinPrice:                 1,
 				ExcessConversionConstant: 5_000,
 			},
+			ValidatorFeeCapacity: 20_000,
+			ValidatorFeeConfig: validatorfee.Config{
+				Target:                   10_000,
+				MinPrice:                 512,
+				ExcessConversionConstant: 51_937_021, // Double every hour
```

Comment:
The formula here btw is: `(Capacity - Target) * NumberOfSecondsPerDoubling / ln(2)` or: `(20000 - 10000) * (60*60) / ln(2)`